ui/pages/confirmations/components/confirm/info/contexts/advanced-details-context.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement, createContext, useContext, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getConfirmationAdvancedDetailsOpen } from '../../../../../../selectors';
+import { setConfirmationAdvancedDetailsOpen } from '../../../../../../store/actions';
+
+type AdvancedDetailsHandlerContextType = {
+  showAdvancedDetails: boolean;
+  setShowAdvancedDetails: (value: boolean) => void;
+};
+
+export const AdvancedDetailsHandlerContext = createContext<
```

Comment:
As this is now just reliant on standard controller state, is it fair to say a React context introduces unnecessary complexity?

We can simply use the selector where we need to read the value, and a dispatch where we want to set it as we do with all controller state?